src/tools/interop/idt/capture/platform/android/android.py

```diff
@@ -134,22 +128,23 @@ def _is_connection_str(adb_input_str: str) -> bool:
     def _check_connect_wireless_adb(self, temp_device_id: str) -> None:
         if Android._is_connection_str(temp_device_id):
             connect_command = f"adb connect {temp_device_id}"
-            print(
+            self.logger.warning(
                 f"Detected connection string; attempting to connect: {connect_command}")
-            Bash(connect_command, sync=True, capture_output=False)
+            Bash(connect_command, sync=True, capture_output=False).start_command()
```

Comment:
It would be nice for the `Bash` wrapper to provide return status for synchronous commands, and you should probably be checking the return status, unless you plan on allowing everything to blowup and then be cleaned up

Reply:
There is `finished_success()` already in this class which provides the same functionality for sync and async commands. My goal with this class was to make the behavior for background and foreground somewhat uniform and I don't like have a function with multiple return types. Would rather have a variable with multiple types (self.proc) be managed internally and make it uniform through the instance methods, so to do that start can't return code for sync.